### DISCUSSION TITLE
[Feature] Add to album from search request

### DISCUSSION THREAD

**Discussion by mr-biz** [immich-app/immich#9244] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I go to several locations per day to take 500+ asset photos. I can search by date and location but currently cannot add the search results to an album.  So an add to album button on the search results would be very nice. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
If you select assets in the search result view, the add to album button is in the top right.

**Reply by mr-biz** [reply]
I tried again.  There is no add-to album button on top of RHS.
I am using v.1.103.1.  I also tried clearing browsing data.
![image](https://github.com/immich-app/immich/assets/44796116/3c4e8385-8382-4417-be0a-767876d697b8)
If I select a single image, I can add to an album but there is no way to add all photos from the search criteria to an album

**Reply by bo0tzz** [reply]
After you select an image, next to the add to album button there will be a select all button that you can use.

In the future, your need will be covered by #1673

**Comment by mr-biz** [comment]
Got it.  Thank you for your help, it is much appreciate.

On Sat, 4 May 2024 at 12:36, bo0tzz ***@***.***> wrote:

> After you select an image, next to the add to album button there will be a
> select all button that you can use.
>
> In the future, your need will be covered by #1673
> <https://github.com/immich-app/immich/discussions/1673>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/immich-app/immich/discussions/9244#discussioncomment-9313163>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKVYRVDTR5SD6XRDEQREWFLZATB35AVCNFSM6AAAAABHFZ3RBOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TGMJTGE3DG>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


Online ESL Teacher

Email




Tel


***@***.***



+44 20 3286 3334

+44 742 602 7024
